spdx-utils/src/main/resources/licenserefs/LicenseRef-scancode-996-icu-1.0

```diff
@@ -0,0 +1,44 @@
+"Anti 996" License Version 1.0 (Draft)
```

Comment:
Commit message nit: It would have been nice to remind us *how* the update is performed, similar to like you explained it in the other commit's message.

Reply:
Done.

Reply:
Another nit: We usually use refernces for things like URLs, but not for code. I would have expected something like:

```
By using:

$ ./gradlew :spdx-utils:generateLicenseRefTextResources
```

Reply:
Same for the next commit, BTW.

Reply:
I believe the commit message is good enough in this aspect.

Reply:
I've updated it regardlessly.